src/lib.rs

```diff
@@ -12,12 +12,12 @@ use hdk::prelude::*;
 pub mod defs;
 pub use defs::*;
 
-const ROOT_ANCHOR: &'static str = concat!("holochain_anchors", "::", "root_anchor");
+pub const ROOT_ANCHOR: &'static str = concat!("holochain_anchors", "::", "root_anchor");
```

Comment:
This should not be public, There's no reason to ever use ROOT_ANCHOR outside this crate. It's used internally and checked that it exists before use. Exposing it would invalidate these checks.